src/qml/pages/stub.qml

```diff
@@ -16,16 +17,21 @@ ApplicationWindow {
 
     Component.onCompleted: initExecutor.initialize()
 
-    Connections {
-        target: initExecutor
-        onInitializeResult: nodeModel.initializeResult(success, tip_info)
-    }
-
+    Loader {
+        id: loader
+        active: initExecutor.ready
+        anchors.fill: parent
+        sourceComponent: Item {
+            NodeModel {
+                id: node_model
+            }
```

Comment:
Does instantiating an object in QML have performance penalties comparing to instantiating it in C++ code?